docs/configure/ci/configuration.md

```diff
@@ -73,3 +73,42 @@ It's a good practice to include the `reason` field for future reference.
 * `exemptions[].report` - String - the name of the report type (`polaris`, `pluto`, `trivy` or `opa`)
 * `exemptions[].checks[]` - Array - an array of check IDs to skip (e.g. `runAsNonRoot`)
 * `exemptions[].reason` - String - a human-readable description of why this exemption is necessary
+
+## Troubleshooting and Advanced Configurations
+
+### Git Checkouts
+Insights needs to gather the following information from Git:
+* The hash of the current commit
+* The hash of the base commit
+* The commit message
+* The branch name
+* The origin URL
+
+Some CI providers only provide a partial checkout of the Git repository
+by default, and some (e.g. Google Cloud Build) check out the repository
```

Comment:
```suggestion
by default and some (e.g. Google Cloud Build) check out the repository
```

Reply:
I disagree 😄 but I generally have a bias for punctuation

Reply:
Yeah you totally love the ","
I obviously don't have a strong opinion here haha